cosmwasm/enclaves/shared/contract-engine/src/contract_validation.rs

```diff
@@ -859,7 +860,7 @@ fn verify_callback_sig_impl(
 
 #[allow(clippy::too_many_arguments)]
 fn verify_input_params(
-    sig_info: &SigInfo,
+    #[cfg(feature = "light-client-validation")] sig_info: &SigInfo,
```

Comment:
Is this needed for the SNIP-52 stuff? If not I suggest adding it as a different PR

Reply:
It's not, it's just to stop the compiler from complaining that this is not used when this flag is off.

Reply:
https://github.com/scrtlabs/SecretNetwork/pull/1506